x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_routing/review_suggestions_form/create_stream_confirmation_modal.tsx

```diff
@@ -20,28 +21,28 @@ import {
   EuiFieldText,
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import React from 'react';
 import type { Streams } from '@kbn/streams-schema';
 import { type ReviewSuggestionsInputs } from './use_review_suggestions_form';
-import { useForkStream } from './use_fork_stream';
 import { ConditionPanel } from '../../shared';
+import {
+  useStreamRoutingEvents,
+  useStreamsRoutingSelector,
+} from '../state_management/stream_routing_state_machine';
 
 export function CreateStreamConfirmationModal({
-  definition,
   partition,
-  onClose,
-  onSuccess,
 }: {
   definition: Streams.WiredStream.GetResponse;
   partition: ReviewSuggestionsInputs['suggestions'][number];
-  onClose: () => void;
-  onSuccess: () => void;
```

Comment:
Removing the onSuccess, the suggestions are not removed anymore once accepted, they stay on the list.

Reply:
Fixed